omod/src/main/webapp/pages/home.gsp

```diff
@@ -1,34 +1,39 @@
 <%
     ui.decorateWith("appui", "standardEmrPage", [ title: ui.message("referenceapplication.home.title") ])
+    ui.includeCss("referenceapplication", "home.css")
```

Comment:
is it `home.css` or `home.scss`?

Reply:
it is css normally referring after the build.